mirage/config.js

```diff
@@ -1,7 +1,9 @@
 export default function() {
   this.get('/reminders');
   this.post('/reminders');
+  this.patch('/reminders');
```

Comment:
I don't think you need this line - you are only patching an individual reminder. Have you verified that this is necessary?